freqtrade/rpc/rpc_manager.py

```diff
@@ -15,43 +14,31 @@ class RPCManager(object):
     Class to manage RPC objects (Telegram, Slack, ...)
     """
     def __init__(self, freqtrade) -> None:
-        """
-        Initializes all enabled rpc modules
-        :param config: config to use
-        :return: None
-        """
-        self.freqtrade = freqtrade
-
-        self.registered_modules: List[str] = []
-        self.telegram: Any = None
-        self._init()
+        """ Initializes all enabled rpc modules """
+        self.registered_modules: List[RPC] = []
 
-    def _init(self) -> None:
-        """
-        Init RPC modules
-        :return:
-        """
-        if self.freqtrade.config['telegram'].get('enabled', False):
+        # Enable telegram
+        if freqtrade.config['telegram'].get('enabled', False):
             logger.info('Enabling rpc.telegram ...')
-            self.registered_modules.append('telegram')
-            self.telegram = Telegram(self.freqtrade)
+            from freqtrade.rpc.telegram import Telegram
+            self.registered_modules.append(Telegram(freqtrade))
 
     def cleanup(self) -> None:
-        """
-        Stops all enabled rpc modules
-        :return: None
-        """
-        if 'telegram' in self.registered_modules:
-            logger.info('Cleaning up rpc.telegram ...')
-            self.registered_modules.remove('telegram')
-            self.telegram.cleanup()
+        """ Stops all enabled rpc modules """
+        logger.info('Cleaning up rpc modules ...')
+        for mod in self.registered_modules:
+            logger.debug('Cleaning up rpc.%s ...', mod.name)
+            mod.cleanup()
+
+        self.registered_modules = []
```

Comment:
i'm still thinking if this could introduce a potential cleanup-bug/leak in the future (if more than telegram exists)

maybe something like the below ... 
``` python
while self.registered_modules:
    mod = self.registered_modules.pop()
    mod.cleanup()
    del mod 
```

i guess it could eventually (if ctrl+c is pressed multiple times) - but I'm not sure and it's currently difficult to test as this list can only have 1 item so far.

looking for other opinions on this - don't take the above for granted :)